training/provision/templates/Containerfile.j2

```diff
@@ -0,0 +1,9 @@
+FROM quay.io/ai-lab/{{ image_name }}:latest
```

Comment:
`podman build` already has `--from` to template this without writing a new containerfile btw.